### PR TITLE
ci(debug-test): add debug lines to aide with debugging

### PR DESCRIPTION
**Pull request by padlar**

### CODE REVIEW

File: internal/controller/stas/scan_job_controller.go [hunk 1]
```diff
@@ -114,6 +115,7 @@ func (r *ScanJobReconciler) reconcileCompleteJob(ctx context.Context, jobName st
 		cis.Status.LastScanTime = &now
 		cis.Status.LastScanJobName = jobName
 		err = r.Status().Patch(ctx, cis, client.MergeFrom(cleanCis))
+		logf.FromContext(ctx).V(3).Info("Patched CIS status with reason: %q patchStatus: %s\n", condition.Reason, err)
```
Comment (erikgb) [inline internal/controller/stas/scan_job_controller.go:118]: Should we include the CIS name?

File: internal/controller/stas/scan_job_controller.go [hunk 2]
```diff
@@ -169,6 +171,8 @@ func (r *ScanJobReconciler) reconcile() reconcile.Func {
 func (r *ScanJobReconciler) reconcileJob(ctx context.Context, job *batchv1.Job) error {
 	cisList := &stasv1alpha1.ContainerImageScanList{}
 
+	logf.FromContext(ctx).V(3).Info("Reconciling job(%s) with status %+v", job.Name, job.Status)
```
Comment (erikgb) [inline internal/controller/stas/scan_job_controller.go:174]:
```suggestion
	logf.FromContext(ctx).V(3).Info("Reconciling job %q with status %+v", job.Name, job.Status)
```

And what does the `+` in `%+v` do?

File: internal/controller/stas/scan_job_controller.go [hunk 3]
```diff
@@ -197,6 +201,7 @@ func (r *ScanJobReconciler) reconcileJob(ctx context.Context, job *batchv1.Job)
 	if err != nil {
 		switch {
 		case staserrors.IsJobPodNotFound(err), staserrors.IsScanJobContainerWaiting(err):
+			logf.FromContext(ctx).V(3).Info("Failed to get logs %s", err)
```
Comment (erikgb) [inline internal/controller/stas/scan_job_controller.go:204]: Include job name?

File: internal/controller/stas/scan_job_controller.go [hunk 4]
```diff
@@ -211,8 +216,10 @@ func (r *ScanJobReconciler) reconcileJob(ctx context.Context, job *batchv1.Job)
 	}(logs)
 
 	if job.Status.Succeeded > 0 {
+		logf.FromContext(ctx).V(3).Info("Patching CIS status with success job status")
```
Comment (erikgb) [inline internal/controller/stas/scan_job_controller.go:219]: Include CIS name?

File: internal/controller/stas/scan_job_controller.go [hunk 5]
```diff
@@ -211,8 +216,10 @@ func (r *ScanJobReconciler) reconcileJob(ctx context.Context, job *batchv1.Job)
...
 		return r.reconcileCompleteJob(ctx, job.Name, logs, cis)
 	} else {
+		logf.FromContext(ctx).V(3).Info("Patching CIS status with failed job status")
```
Comment (erikgb) [inline internal/controller/stas/scan_job_controller.go:222]: Include CIS name?

**Review by erikgb — CHANGES_REQUESTED** [review]
Some comments/questions.

File: internal/controller/stas/scan_job_controller.go (hunk 2 above)
Comment (padlar) [reply internal/controller/stas/scan_job_controller.go]: when printing structs, the plus flag adds field names

File: internal/controller/stas/scan_job_controller.go (hunk 4 above)
Comment (padlar) [reply internal/controller/stas/scan_job_controller.go]: included

File: internal/controller/stas/scan_job_controller.go (hunk 1 above)
Comment (padlar) [reply internal/controller/stas/scan_job_controller.go]: included

File: internal/controller/stas/scan_job_controller.go [hunk 6]
```diff
@@ -78,6 +78,7 @@ func (r *ScanJobReconciler) reconcileCompleteJob(ctx context.Context, jobName st
 		cis.Status.LastScanJobName = jobName
 		err = r.Status().Patch(ctx, cis, client.MergeFrom(cleanCis))
 
+		logf.FromContext(ctx).V(3).Info("Patched CIS status with reason: %q patchStatus: %s\n", condition.Reason, err)
```
[the remark this answers is not in the document]
Comment (padlar) [reply internal/controller/stas/scan_job_controller.go]: included

File: internal/controller/stas/scan_job_controller.go (hunk 3 above)
Comment (padlar) [reply internal/controller/stas/scan_job_controller.go]: included

File: internal/controller/stas/scan_job_controller.go (hunk 5 above)
Comment (padlar) [reply internal/controller/stas/scan_job_controller.go]: included

**Review by erikgb — APPROVED** [review]
lgtm

**Pull request merged**